index.html

```diff
@@ -1052,6 +1097,160 @@ <h4 id="Abilities">Abilities</h4>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>Legendary Power
+               </td>
+               <td>
+                  Light / Medium / Heavy values increase by 1 / 2 / 3 respectively.
+               </td>
+               <td>Can be taken up to 3 times. You may only take either Legendary Endurance or Legendary Power
+                  abilities a total of 3 times. Such as taking Legendary Endurance 3 times and not taking Legendary Power (or vice versa),
+                  or 2 Legendary Endurance and 1 Legendary Power (or vice versa).
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Empowered Resistance Drain
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Resistance Drain. When you use Empowered Resistance Drain, the number
+                  of resistance points spent grants another creature of your choice bonus damage on the next instance of damage they 
+                  inflict before the start of your next turn. 1, 2 or 3 resistance points spent results in additional Light, Medium, or Heavy damage
+                  respectively - the type of damage is the same as what the attack deals.
+                  </p>
+                  Any creature can only give this effect once per turn, and any creature can only have one instance of this effect on
+                  them at any given time. 
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Empowered Ability
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Empowered Ability. When you use Empowered Ability, the number
+                  of resistance points spent grants additional advantage (+) equal to the number of resistance points spent, 
+                  or grants additional damage equal to the number of resistance points spent - your choice. The type of damage 
+                  is the same as what the attack deals. If the ability deals no damage it can still deal this bonus damage of the 
+                  most appropriate type - GM's discretion.
+               </td>
+               <td>Can be taken once.
+               </td>
+            </tr>
+            <tr>
+               <td>
+                  Legendary Heroic Resistance
+               </td>
+               <td>
+                  <p>
+                  Upgrade to Heroic Resistance. On your turn for 0 AP you may spend any number of resistance points of any type.
```

Comment:
On the flip side, this feels incredibly strong.   As a brawler type going SPI build spending one resistance point a turn feels pretty cheap and can keep it going throughout the entire fight.   If you'd expect to do legendary resistance one every 4 turns (which isn't that often in a boss fight as a frontliner) you'd absolutely just have this constantly popping.